### ISSUE TITLE
DSi Mode failing to load any dsiware

### ISSUE THREAD

**Issue by Bonboon229** [Arisotura/melonDS#1219]
Attempting to "load any DSi software, or even DS games in DSi mode causes the "An Error Has Occurred" Screen to pop up.

**Comment by Arisotura** [comment]
you probably have bad ROM dumps

**Comment by Bonboon229** [comment]
I can't even access DSi Settings

**Comment by RayyanAnsari** [comment]
what version are you on and on what version did it break?

**Comment by Bonboon229** [comment]
Using the latest version from the Actions TAB, and I don't know when it broke, as I only recently updated from the working version

**Comment by RayyanAnsari** [comment]
try before and after 'Misc fixes for dsiwifi BMI stage'.

**Comment by Bonboon229** [comment]
okay, gimme a few mins

**Comment by Bonboon229** [comment]
Yeah, the release after 'Misc fixes for dsiwifi BMI stage' is when things broke

**Comment by RayyanAnsari** [comment]
do you have unlaunch installed?

**Comment by Bonboon229** [comment]
No, no homebrew worked for me. any homebrew threw up the "Error has Occured" Screen on launch. Unlaunch included.

**Comment by RayyanAnsari** [comment]
even before that commit?
(could you join us on [IRC/Discord](http://melonds.kuribo64.net/board/thread.php?id=3) for a faster conversation?

**Comment by Bonboon229** [comment]
Even Before. And yeah, I'll join the discord.

**Comment by RayyanAnsari** [comment]
for future reference,
https://github.com/Arisotura/melonDS/pull/1216#issuecomment-914396850
